### ISSUE TITLE
x/crypto: tag v1.0.0

### ISSUE THREAD

**Issue by FiloSottile** [golang/go#56325]
x/ repo tagging is here!

@heschi mentioned in #48523 that the system supports v1 as well, and I think that's a better fit for x/crypto than v0. The API is absolutely full of things I wish were not there, but it's been around for too long, and I don't think it would be tenable to make any backwards-incompatible changes without bumping the import path (which going from v0 to v1 doesn't do).

In practice, we've been upholding the Go Compatibility Promise in x/crypto for years, so we're effectively already at v1, we just need to call it that.

/cc @golang/security

**Comment by dmitshur** [comment]
I agree that x/crypto behaves as if it’s already tagged v1.0.0, both when this was filed and now, it just hasn’t been tagged v1 yet, and it should. This issue hasn’t been progressing as is, and importantly once that tag is made, the only way to get back to v0 is to start a new module. So this should probably go through the proposal process.

**Comment by rolandshoemaker** [comment]
The argument against doing this is that we've spent the last few years trying to move things from x/crypto into the standard library. Tagging v1 may give people a false impression about the status of the module. I think ideally we'd tag v1 once we've either moved or frozen everything in x/crypto, and there is essentially nothing left to do.
